src/modules/jcms_rest/src/EventSubscriber/ExceptionJcmsJsonSubscriber.php

```diff
@@ -19,15 +19,6 @@ protected function getHandledFormats() {
     return ['jcms_json'];
   }
 
-  /**
-   * {@inheritdoc}
-   */
-  protected static function getPriority() {
-    // This will fire after the most common HTML handler, since HTML requests
-    // are still more common than JSON requests.
-    return -75;
```

Comment:
Since updating to `8.5.1` this was not being prioritized highly enough. It fixed the issue if I removed it and relied upon the default priority.